_website-config-docs.json

```diff
@@ -344,6 +344,45 @@
         "ReplaceKeyPrefixWith": null,
         "ReplaceKeyWith": "docker-enterprise/v3.0/dockeree-products/index.html"
       }
+    },
+    {
+      "Condition": {
+        "HttpErrorCodeReturnedEquals": null,
+        "KeyPrefixEquals": "v17.06/"
+      },
+      "Redirect": {
+        "HostName": "docs.docker.com",
+        "HttpRedirectCode": null,
+        "Protocol": "https",
+        "ReplaceKeyPrefixWith": "",
+        "ReplaceKeyWith": null
+      }
+    },
+    {
+      "Condition": {
+        "HttpErrorCodeReturnedEquals": null,
+        "KeyPrefixEquals": "v18.03/"
+      },
+      "Redirect": {
+        "HostName": "docs.docker.com",
+        "HttpRedirectCode": null,
+        "Protocol": "https",
+        "ReplaceKeyPrefixWith": "",
+        "ReplaceKeyWith": null
+      }
+    },
+    {
+      "Condition": {
+        "HttpErrorCodeReturnedEquals": null,
+        "KeyPrefixEquals": "v18.09/"
+      },
+      "Redirect": {
+        "HostName": "docs.docker.com",
+        "HttpRedirectCode": null,
+        "Protocol": "https",
```

Comment:
I saw in the docs that these are optional, so thinking of doing a follow-up to test if redirects work ok without these specified (in that case, we could possibly use a single config both for `docs-stage` and for `docs`

Reply:
I think I tried that, but maybe I did it wrong.
Anyhow the feedback loop is 
1. create PR
2. merge PR to master
3. test redirects on docs-stage.docker.com
4. revert PR if it broke something

Reply:
The other idea I had before adding two json files was, we could specify the HostName value when running the GitHub action https://github.com/docker/docker.github.io/blob/master/.github/actions/update-website-config/main.go that uploads the website config in the workflow https://github.com/docker/docker.github.io/blob/master/.github/workflows/build-master.yml#L27-L32

Reply:
I think I remember, when you leave that HostName away, then the internal URL to the S3 bucket is used after the redirect without the docker.com domain.

Reply:
Ah, that'd make sense yes, that's a pity